solidity/scripts/get-prices.js

```diff
@@ -1,6 +1,4 @@
-const fs = require("fs")
-const path = require("path")
-const Web3 = require("web3")
+const web3 = require("web3")
```

Comment:
The capitalization change here confuses me… Was it broken before? O.o

Reply:
That one was weird to me too. Further down in the file we have:
```
      const value = await web3.eth.getStorageAt(address, 1)
```
with no other reference to `web3` in the file other than the `Web3` import. Do you know if the capitalization is irrelevant?

Reply:
The reason i changed it is that the linter was saying that `Web3` was unused!